docs/specification.md

```diff
@@ -446,7 +448,10 @@ Parameter (in Michelson):
 ```
 (pair
   (address %to_)
```

Comment:
Why do we have `from` in `burn`, but `to_` (with `_`) in `mint`?

Reply:
I've changed to use `from_` instead now.